File: src/element.c
#include "../head/element.h"
#include <stdlib.h>
#include <stdio.h>
#include <string.h>

void affiche_ELEMENT(ELEMENT element)
{
      char commande[1000];
	  printf( "id = %s \n", element.id);
      printf( "nom fichier = %s \n", element.nom_fic);
	  strcpy(commande, "wc -l ");
	  strcat(commande, element.CHEMIN); 
	  strcat(commande, element.nom_fic);
	  system(commande);
	  printf("\n==================================\n");
}

void affect_ELEMENT(ELEMENT *elemaffect, ELEMENT e2)
{
    *elemaffect=e2;
}

File: JAVA/terminal_version/Entite/Element.java
package Entite;

public class Element {
    public String ID;
    public String chemin;

    public Element (String ID, String chemin)
    {
        this.ID=ID;
        this.chemin=chemin;
    }
    public String toString()
    {
        String text="";
        text="ID: "+ID+"\n Chemin: "+chemin;
        return text;
    }

}
